packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -3,23 +3,81 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import QuickLinks from './components/quick-links/QuickLinks';
 import NavItems from './components/nav-items/NavItems';
+import NavHelp from './components/nav-help/NavHelp';
+import UserProfile from './components/user-profile/UserProfile';
+import OutlineInfo from './icons/OutlineInfo';
+import UserIcon from './icons/UserIcon';
+import OutlineEllipses from './icons/OutlineEllipses';
 import styles from './Nav.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
   test: PropTypes.string,
-  help: PropTypes.string,
+  help: PropTypes.arrayOf(PropTypes.object),
   isMobileNavOpen: PropTypes.bool, // this is proof of concept. should we keep this?
   quickLinks: PropTypes.arrayOf(PropTypes.object),
   navItems: PropTypes.arrayOf(PropTypes.object),
+  profile: PropTypes.object,
```

Comment:
Please use `Proptypes.shape` to validate the proper object structure is passed https://facebook.github.io/react/docs/typechecking-with-proptypes.html